app/Infrastructure/Cache/WalletCache.php

```diff
@@ -36,7 +36,7 @@ public function set(string $wallet_id, Wallet $wallet): void
         Cache::put('wallet'.$wallet_id,$wallet,600);
     }
 
```

Comment:
Maybe the name of the variable is better if we make the following change:
$wallet_id ---> $walletId